### PR TITLE
Mita' year book

### PR DESCRIPTION
**Pull request by Mita-cpu**
commit

### CODE REVIEW

File: 2021/yearbook.html [hunk 1]
```diff
@@ -32,6 +32,13 @@
     <h2>2021 Software Development Engineer Bootcamp</h2>
 
     <div class="yearbook">
+           <a>
+       <a href="https://mita-cpu.github.io/" target="_blank" />
+      <figure>
+          <img src="images/my-picture.jpg" alt="Mita" />
```
Comment (anthonygraignic) [inline 2021/yearbook.html]: Please create a `mita.jpg` file and restore other deleted files `nothemba.jpg`, `anthony.jpg`

File: index.html [hunk 2]
```diff
@@ -46,17 +46,15 @@
...
+      <p>Girlcode Learnership</p>
       <div style="margin: 24px 0">
-        <a href="https://twitter.com/GirlCode_za"><i class="fa fa-twitter"></i></a>
-        <a href="https://www.linkedin.com/company/girlcodeza"><i class="fa fa-linkedin"></i></a>
-        <a href="https://github.com/GirlCodeZA"><i class="fa fa-github"></i></a>
+        <a href="https://twitter.com/Mita_Raboya"><i class="fa fa-twitter"></i></a>
       </div>
       <p>
-        <a href="mailto:girlcode@gmail.com"><button>Contact</button></a>
+        <a href="mailto:mitaraboya@gmail.com@gmail.com"><button>Contact</button></a>
```
Comment (anthonygraignic) [inline index.html]: This file should not be changed, please revert.

Your changes should only contain:
- your profile pic in 2021/images
- the HTML block in the `yearbook.html` that you correctly did 

I would advise you start from scratch after saving your jpg file and your HTML code to add in order to make a clean pull request with the only required changes.

**Review by anthonygraignic — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (anthonygraignic) [reply index.html]: You are also there !!

File: 2021/yearbook.html (hunk 1 above)
Comment (Mita-cpu) [reply 2021/yearbook.html]: i have created a mita.jpg

**Comment by anthonygraignic** [comment]
Merged in #32

**Pull request closed without merging**